src/hawtio/plugins/shared/workspace.ts

```diff
@@ -31,10 +31,11 @@ class Workspace {
       }
     }
     const value = await jolokiaService.list(options)
+
     // TODO: this.jolokiaStatus.xhr = null
     const domains = this.unwindResponseWithRBACCache(value)
     log.debug("JMX tree loaded:", domains)
-    return new MBeanTree(domains)
+    return MBeanTree.createMBeanTreeFromDomains('workspace', domains)
```

Comment:
We use the naming convention `hawtio-<plugin>-<area>` for loggers. Let's follow the convention.

Reply:
Not sure that this necessarily works for the workspace anymore since it is outside the plugins. As a singleton, it is always available as not tied to one plugin. Would `hawtio-plugin-workspace` or `hawtio-shared-workspace` work?

Reply:
Originally it was `hawtio-jmx-workspace` and `hawtio-jmx-workspace-tree`. You can easily see the original set of logger names by launching the legacy hawtio SB example and then looking at the console logs preferences and hitting the `add` button. For this case, `hawtio-shared-workspace` looks good.